ModuleChangelog.md

```
@@ -1,3 +1,43 @@
+## 2.15.0
+Release Date: June 30, 2026
+
+#### RELEASE NOTES
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)